src/test/users.test.ts

```diff
@@ -0,0 +1,59 @@
+//import { expect } from 'chai';
+import { AppDataSource } from '../data-source';
+import { User } from '../entity/User';
+import { queryGetAllUsers } from './query-users';
+import { expect } from 'chai';
+import { newUser } from '../utils/data-generate-users';
+import { generateFakeToken } from './functions';
+
+describe('query users', async () => {
+  const users = [];
+  const token: string = await generateFakeToken();
+
+  beforeEach(async () => {
+    for (let i = 0; i < 20; i++) {
+      users.push(await newUser());
+    }
+
+    AppDataSource.getRepository(User).save(users);
+  });
+
+  afterEach(async () => {
+    AppDataSource.getRepository(User).delete({});
+  });
+
+  it('should return a vector of users', async () => {
+    const formatArray = users
+      .map((user) => {
+        const newUser = {
+          id: user.id,
+          name: user.name,
+          email: user.email,
+          birthDate: user.birthDate
+        };
+        return newUser;
+      })
+      .sort((a, b) => a.name.localeCompare(b.name));
+
+    const response = await queryGetAllUsers(token, 1, 20);
+
+    const listUsersResponse = response.data.data.users;
+    const objTest = {
```

Comment:
```suggestion
    const expectedResponse = {
```

Aqui tbm, ponto semelhante ao comment anterior.